packages/backend-common/src/middleware/errorHandler.ts

```diff
@@ -81,8 +81,7 @@ export function errorHandler(
     }
 
     res.status(status);
-    res.setHeader('content-type', 'text/plain');
-    res.send(message);
+    res.json(message);
```

Comment:
This is not json (yet - we will be working to make this a better error conveying system later) - should revert this and the test for now